Adopt B95 SoC

s07641069 opened the pull request:
Adaptation for B95 SoC with correct HAL drivers and libs

Merged PRs:
https://github.com/telink-semi/zephyr/pull/221
https://github.com/telink-semi/zephyr/pull/235

andriy-bilynskyy commented:
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| hal_telink | https://github.com/telink-semi/hal_telink/commit/0f03ecc893bd5818338ebccf860474310099905e | https://github.com/telink-semi/hal_telink/commit/92c7d939eb086408d07e470eae167fa65e45fa6e (develop) | [telink-semi/hal_telink@0f03ecc8..92c7d939](https://github.com/telink-semi/hal_telink/compare/0f03ecc893bd5818338ebccf860474310099905e..92c7d939eb086408d07e470eae167fa65e45fa6e) |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

andy-liu-telink committed:
soc: drivers: boards: telink_b95: Update drivers

- Update Telink B9x SOC due to B95 chip
- Update tlsr9258a board based on B95 chip
- Extend Telink B9x drivers with B95 chip
- Update Telink HAL

Co-authored-by: Zhenghuan Zhang <zhenghuan.zhang@telink-semi.com>
Co-authored-by: Lingzhi Chen <lingzhi.chen@telink-semi.com>
Co-authored-by: Haiwen Xia <haiwen.xia@telink-semi.com>

Signed-off-by: Andy Liu <jie.liu@telink-semi.com>
